docs/Rollup_basics.md

```diff
@@ -63,11 +66,14 @@ That's what makes ArbChains trustless.
 At this point, you might be wondering who will be validators of your ArbChain.
 Arbitrum allows anyone to validate any ArbChain, but who will step up and do it?
 
-As a dapp developer, you would probably want to validate your dapp's ArbChain, to help ensure its correctness and show confidence in your dapp.
-Some users will want to validate as well, in order to protect their interest in the dapp's correct execution.
+Although validation is permissionless and anyone _can_ validate, it's important to make sure that each ArbChain always has one honest validator that is paying attention. As a dApp developer, you would probably want to validate your dApp's ArbChain, to help ensure its correctness and show confidence in your dApp.
+
+For developers who do not want to validate themselves, you'll be able to hire a validator-as-a-service to validate your ArbChain once Arbitrum is released for production.  
+You might want to hire more than one, if you or your users worry that one might misbehave. As long as any validator behaves honestly, the Arbitrum protocol guarantees that all contracts in the ArbChain will run correctly according to their code. 
+
+Some users will want to validate as well, in order to protect their interest in the dapp's correct execution. A healthy ArbChain with an engaged userbase will likely have a variety of validators.
```

Comment:
After the "Some users will want" sentence, add one saying that users can hire validators if they want.